contrib/tools/workflowcheck/workflow/checker.go

```diff
@@ -176,7 +180,17 @@ func (c *Checker) Run(pass *analysis.Pass) error {
 			}
 			// Report if couldn't get type
 			if fn == nil {
-				pass.Reportf(callExpr.Args[0].Pos(), "unrecognized function reference format")
+				_, isIgnored := ignoreMap[n]
```

Comment:
I think a better impl is to change `if callExpr == nil {` above to `if callExpr == nil || isIgnored {`. You are ignoring `RegisterWorkflow` call so check it early.